Point to the correct type declaration file in package.json

kroy opened the pull request:
Super minor quality of life improvement for typescript enjoyers. VSCode wasn't finding the type declarations in `types/index.d.ts` because of the missing `.d.` in the type dec filename.

Tested by linking locally w/ my fix, and intellisense was much happier.

wlamason commented:
@kroy thanks for catching that! Merging your PR now. 

Thank you for contributing and being a part of open source 🙂

wlamason commented:
@kroy The most recent version 2.0.3 with these changes should now be available on [npm](https://www.npmjs.com/package/opendota.js). 
https://www.npmjs.com/package/opendota.js

As always, if you find this package useful, consider leaving a ⭐ in the top right of the repository as it helps others find the package through increasing its popularity 🙂 

Enjoy!

kroy commented:
@wlamason I unfortunately was not a very good citizen here and changed the wrong filename 🙈 Something must have gotten mixed up while I was testing things. I fixed in https://github.com/wlamason/opendota.js/pull/21, but understand if you just want to revert and move on w/ your life.

Thanks for your time!
